### ISSUE TITLE
 webhooks

### ISSUE THREAD

**Issue by tongalalal** [allegro/allegro-api#9624]
Does the platform have a mechanism similar to webhooks, where if a buyer cancels an order, it can send a request to our callback interface to provide information about the canceled order and the seller? Is there such an interface available?

**Comment by AureliuszBrussy** [comment]
No, we do not support webhooks currently.

**Comment by Toumash** [comment]
No, but theres a event stream https://developer.allegro.pl/documentation#operation/getOrderEventsUsingGET
`/order/events` which you can poll events from
